src/Step/ValidateConfiguration.php

```diff
@@ -47,19 +47,20 @@ public function execute(): void
     {
         $this->logger->notice('Validating configuration');
 
-        $messages = $this->collectMessages();
+        $errors = $this->collectErrors();
 
-        ksort($messages);
-        foreach ($messages as $level => $levelMessages) {
-            $error = 'Fix configuration with given suggestions:' . PHP_EOL . implode(PHP_EOL, $levelMessages);
+        ksort($errors);
+        foreach ($errors as $level => $levelErrors) {
```

Comment:
It will be easier to find where the methods of Error object is used.
```suggestion
         /** @var Error[] $levelErrors */
        foreach ($errors as $level => $levelErrors) {
```

Reply:
added